packages/hub/services/mailchimp.ts

```diff
@@ -0,0 +1,57 @@
+// the types from Definitely Typed are not good enough.
+// Mailchimp docs seem more reliable though the examples have the wrong node package name
+// https://mailchimp.com/developer/marketing/api
+// @ts-ignore
+import client from '@mailchimp/mailchimp_marketing';
+import config from 'config';
+
+export default class Mailchimp {
+  client = client;
+  newsletterListId = config.get('mailchimp.newsletterListId') as string;
```

Comment:
This will fail if mailchimp.newsletterListId is not defined. That is good for staging/production, but do we really care on development?

Reply:
I don't think having the API key missing and as such Mailchimp failing matters as much during development (there is a default `newsletterListId`), but feel hesitant to add a separate flow that can fail silently. Is there a good way to do this?

Reply:
We should fail on/near boot if we don't have an API Key defined and we're in staging or prod. Probably can do this in the constructor of the service.

Reply:
This is done in the latest commit. Any missing essential config will cause an Error during initialization of the service in non-development environments.